src/videojs-hls.js

```diff
@@ -392,6 +392,37 @@ videojs.Hls.prototype.addCuesForMetadata_ = function(segmentInfo) {
   minPts = Math.min(isFinite(segment.minVideoPts) ? segment.minVideoPts : Infinity,
                     isFinite(segment.minAudioPts) ? segment.minAudioPts : Infinity);
 
+  Cue = window.WebKitDataCue || window.VTTCue;
+
+  deprecateOldCue = function(cue) {
+    if (cue.frame.id) {
+      Object.defineProperty(cue.frame, 'id', {
```

Comment:
It looks like these statements are overwriting the values at `cue.frame.whatever` with getters that log a warning and return `undefined`. Am I reading that correctly?

I thinking make the old properties still usable (for now), but log a warning. More like:
```js
Object.defineProperty(cue.frame, 'id', {
  get: function() {
    videojs.log.warn('cue.frame.id is deprecated. Use cue.value.key instead.');
    return cue.value.key;
  }
});
```